docs/tutorial.md

```diff
@@ -24,33 +37,74 @@ We recommend that your familiarize yourself with basics of event sourcing and CQ
 
 Use the **create-resolve-app** tool to create a new reSolve application:
 
-##### npm:
-
 ```sh
-$ npm i -g create-resolve-app
-$ create-resolve-app shopping-list
+yarn create resolve-app shopping-list
 ```
 
-##### npx:
+After this, a minimal reSolve application is ready. To run it in development mode, type:
 
 ```sh
-$ npx create-resolve-app shopping-list
+cd shopping-list
+yarn dev
 ```
 
-##### yarn:
+### Running Tests
 
-```sh
-$ yarn create resolve-app shopping-list
-```
+A new reSolve application contains a minimalistic [TestCafe](https://testcafe.io/) test located in `test/e2e/index.test.js` that runs on the client and checks that the reSolve server responds normally. You are also free to modify this file to check your application's functionality.
```

Comment:
Is the term "minimalistic" a widely-used term in the context of describing an application/test?
For example, can we say "basic" instead?

Reply:
Thank you.